docs/application/native/guides/multimedia/image-barcode.md

```diff
@@ -107,7 +107,7 @@ To initialize the barcode detection and generation engines for use:
           dlog_print(DLOG_ERROR, LOG_TAG, "error code = %d", error_code);
       ```
 
-      For example, to define the detection target, use the `mv_engine_config_set_int_attribute()` function with the `MV_BARCODE_DETECT_ATTR_TARGET` attribute. The possible values are defined in the `mv_barcode_detect_attr_target_e`enumeration (in [mobile](../../api/mobile/latest/group__CAPI__MEDIA__VISION__BARCODE__MODULE.html#ga47aaef5d40653352c5bee73b227062a6) and [wearable](../../api/wearable/latest/group__CAPI__MEDIA__VISION__BARCODE__MODULE.html#ga47aaef5d40653352c5bee73b227062a6) applications). The default value is `MV_BARCODE_DETECT_ATTR_TARGET_ALL`.
+      For example, to define the detection target, use the `mv_engine_config_set_int_attribute()` function with the `MV_BARCODE_DETECT_ATTR_TARGET` attribute. The possible values are defined in the `mv_barcode_detect_attr_target_e` enumeration (in [mobile](../../api/mobile/latest/group__CAPI__MEDIA__VISION__BARCODE__MODULE.html#ga47aaef5d40653352c5bee73b227062a6) and [wearable](../../api/wearable/latest/group__CAPI__MEDIA__VISION__BARCODE__MODULE.html#ga47aaef5d40653352c5bee73b227062a6) applications). The default value is `MV_BARCODE_DETECT_ATTR_TARGET_ALL`.
```

Comment:
line 110: Add a space. (in the mv_barcode_detect_attr_target_eenumeration)
line 203: Remove this line. (mv_point_s mv_point = {0, 0};)
line 204: Change this line to 'mv_rectangle_s mv_roi = {{0,0}, bardetdata.width, bardetdata.height};'.
line 217: Remove the line. (from the line 217 to 220)
line 226: Remove the line. (from the line 226 to 230)

Reply:
110: No problem. There is a space between mv_barcode_detect_attr_target_e and enumeration.
203:  The original code makes a build error :
```
    mv_rectangle_s mv_roi = {{0,0}, bardetdata.width, bardetdata.height};
```